io.github.trytonvanmeer.DungeonJournal.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "io.github.trytonvanmeer.DungeonJournal",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.32",
+    "runtime-version": "40",
```

Comment:
upgrading runtime requires the "dconf" migration.
See https://docs.flatpak.org/en/latest/sandbox-permissions.html#dconf-access

Reply:
Good catch, fixed